src/main/flight/pid_init.c

```diff
@@ -148,18 +162,40 @@ void pidInitFilters(const pidProfile_t *pidProfile)
             }
             break;
         case FILTER_BIQUAD:
-            pidRuntime.dtermLowpass2ApplyFn = (filterApplyFnPtr)biquadFilterApply;
+            if (pidProfile->dterm_lowpass2_hz < pidFrequencyNyquist) {
+#ifdef USE_DYN_LPF
+                pidRuntime.dtermLowpass2ApplyFn = (filterApplyFnPtr)biquadFilterApplyDF1;
+#else
+                pidRuntime.dtermLowpass2ApplyFn = (filterApplyFnPtr)biquadFilterApply;
+#endif
```

Comment:
Is this needed? D term lowpass 2 isn't dynamic and should just use `biquadFilterApply`

Reply:
Thanks, yes indeed that's not needed here.  Awesome!